src/core/include/openvino/core/model.hpp

```diff
@@ -114,6 +114,9 @@ class OPENVINO_API Model : public std::enable_shared_from_this<Model> {
     /// \brief Clones the original model
     std::shared_ptr<ov::Model> clone() const;
 
+    /// \brief Reference to frontend maanager
+    std::shared_ptr<void> m_femgr;
```

Comment:
It is used to keep frontend manager instance is not released before ov::Model, due to frontend manager has been work as a dynamic instance rather than static instance.

Reply:
In general we don't need to have FR manager here. We need to keep only one frontend which created this particular model. And it already should be done. Here:

https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/core/model.hpp#L46

Reply:
we don't need to keep frontend manager here, ov::Model already keeps a reference to a particular frontend

Reply:
> In general we don't need to have FR manager here. We need to keep only one frontend which created this particular model. And it already should be done. Here:
> 
> https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/core/model.hpp#L46

It cannot work, we will encounter problem in some CI test, sometimes frontend manager will released before ov::Model, which cause segment fault.

Reply:
> we don't need to keep frontend manager here, ov::Model already keeps a reference to a particular frontend

I add such reference at ov::Model is to fix segment fault in CI test failure.

Reply:
> > In general we don't need to have FR manager here. We need to keep only one frontend which created this particular model. And it already should be done. Here:
> > https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/core/model.hpp#L46
> 
> It cannot work, we will encounter problem in some CI test, sometimes frontend manager will released before ov::Model, which cause segment fault.

Can we investigate these issues in order to understand how frontend manager affects `ov::Model`? Frontend manager doesn't create `ov::Model`, it creates frontends and frontend creates `ov::Model`, it means that `ov::Model` depends on frontend but doesn't depend on FE manager.